app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -90,203 +88,84 @@ For safeguarding reasons, schools will assess your suitability to teach from a r
-
-## Information for schools and training providers
-
-### Help from DfE
-
-From 1 February 2023, Ukrainians will be able to obtain QTS through DfE's new professional recognition service,[Apply for QTS in England](#apply-for-qts-in-england).
-
-[Learn more about the new service](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers/a-fairer-approach-to-awarding-qts-to-overseas-teachers--2#). 
-
-### Visas and immigration
-
-Most Ukrainian refugees will not need to be sponsored for a skilled worker or student visa. [Find out more about visa schemes for Ukrainian refugees, allowing them to live, work and study in the UK](#check-the-status-of-your-visa).
-
-### Qualifications
-
-Schools should assess the qualifications and experience of Ukrainian teachers in the same way as other overseas applicants, bearing in mind the exemption provided by the [4 year rule](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
-
-UK ENIC can provide a [statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) to applicants showing how their academic and professional qualifications compare to the English equivalents.
-
-### Alternative roles in school
-
-DfE encourages schools to employ Ukrainian teachers and prospective teachers as teaching assistants if a qualified teacher role isn't available. Ukrainians can then be supported to gain QTS via [assessment only QTS](#assessment-only-qts). 
-
-### Safeguarding
-
-Schools must adopt robust recruitment procedures preventing overseas applicants who are unsuitable to work with children from securing a role in school.
-
-Schools must be guided by the [statutory guidance on safeguarding children and safer recruitment](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2). They should also carry out [additional checks related to international recruitment and training](https://www.gov.uk/guidance/recruit-teachers-from-overseas#makean).  
-
-Given documentation may be difficult for refugees to provide, we expect schools to draw on a varying range of sources on a case by case basis, depending on the individual in question. It is good practice to carry out a risk assessment recording the evidence you have gathered to reach your decision. 
+If you are not in the UK under one of these schemes, [find out about switching to a different type of visa](#visas-giving-you-the-right-to-teach-or-train-to-teach-in-England) so you can train to teach. Not all visas will give you the right to apply for a student loan. 
```

Comment:
Change this URL from:
`#visas-giving-you-the-right-to-teach-or-train-to-teach-in-England`
to:
`#visas-giving-you-the-right-to-teach-or-train-to-teach-in-england`

(path names are lowercase)

Reply:
*lightbulb*
thank you

Reply:
@laurahermionetennant92 If **you're**